src/models/SentimentModel.ts

```diff
@@ -104,6 +103,10 @@ export default class SentimentModel extends BaseModel {
       frame.isImportantAttention = this.calculateImportance(frame.attentionScore, averages.attentionAverage, isImportantTreshold);
       frame.isImportantMood = this.calculateImportance(frame.moodScore, averages.moodAverage, isImportantTreshold);
       frame.isImportantPeople = this.calculateImportance(frame.amountOfPeople, averages.peopleAverage, isImportantTreshold);
+      if (frame.isImportantAttention || frame.isImportantMood || frame.isImportantPeople) {
+        frame.importantFrame = images[imagesIndex].Key;
```

Comment:
if you want to be fancy you can use forEach and access the index directly. https://stackoverflow.com/questions/10179815/get-loop-counter-index-using-for-of-syntax-in-javascript